### ISSUE TITLE
Discuss testing frameworks

### ISSUE THREAD

**Issue by Morwenn** [LoungeCPP/loungesome-cpp#4]
Testing frameworks are a whole thing by themselves. I only ever used Catch but have often heard about Boost.Test, CppUnit and Google Test. Anything you'd like to nominate?

**Comment by nabijaczleweli** [comment]
`bandit`'s pretty dope

**Comment by ghost** [comment]
Yeah, I heard many good things about bandit too, but the syntax is ... special. Maybe it belongs next to catch? who actually uses it?

**Comment by nabijaczleweli** [comment]
It's not "special" it's the other common testing "pattern" (mostly used by FP, ICBWT).

**Comment by Morwenn** [comment]
Looks like Jasmine. Also Catch has started to introduce matchers too even though they're still experimental and not really documented.

**Comment by ghost** [comment]
I hate it, but if you can find a project that actually uses it, I guess it can be nominated.

**Comment by Rerito** [comment]
I've used Google Test a few times. It's quite simple to set up and get going. I just stayed in the basic use cases though so I can't tell the true power of the tool.

**Comment by vendethiel** [comment]
GTest, lol:

https://github.com/google/googletest/issues/863
https://github.com/google/googletest/issues/626
https://chat.stackoverflow.com/transcript/10?m=32483395#32483395

+other stuff
